packages/scene-composer/src/components/panels/scene-rule-components/SceneRuleTargetEditor.tsx

```diff
@@ -55,6 +60,7 @@ export const SceneRuleTargetEditor: React.FC<ISceneRuleTargetEditorProps> = ({
       label: formatMessage(i18nSceneResourceTypeStrings[SceneResourceType[type]]) || SceneResourceType[type],
       value: SceneResourceType[type],
     }));
+  const isAllValid = tagStyle && targetInfo.value === 'Custom';
```

Comment:
isAllValid isn't the clearest variable name.  isCustomStyle or isUsingCustomStyle might be better?

Reply:
+1.

Reply:
Agreed to change in future PR.